stringutils.go

```diff
@@ -70,3 +74,30 @@ func hashsum(s string, hasher hash.Hash) string {
 	hasher.Write([]byte(s))
 	return hex.EncodeToString(hasher.Sum(nil))
 }
+
+func foundInHIBP(s string) int {
+	bytes := []byte(s)
```

Comment:
I would avoid storing the converted value and just call `h.Write([]byte(s))` a few lines below. This isn't a big issue in general, but in this case you're actually overwriting the name `bytes`, which is also the name of a standard library package (and might lead to clashes).